canine/localization/base.py

```diff
@@ -377,8 +377,7 @@ def build_manifest(self, transport: typing.Optional[AbstractTransport] = None) -
             if not transport.isfile(os.path.join(output_dir, '.canine_pipeline_manifest.tsv')):
                 script_path = self.backend.pack_batch_script(
                     'export CANINE_OUTPUTS={}'.format(output_dir),
-                    'cat <(echo "jobId\tfield\tpath") $CANINE_OUTPUTS/*/.canine_job_manifest > $CANINE_OUTPUTS/.canine_pipeline_manifest.tsv',
-                    'rm -f $CANINE_OUTPUTS/*/.canine_job_manifest',
```

Comment:
Don't we also want to clean up the job manifests?

Reply:
Nope, we use them when checking whether we can job avoid to ensure that job outputs are consistent between the task definition and the previous run.

https://github.com/getzlab/canine/blob/99d46732da07b52bc2982b12feb829fc65f4c795/canine/orchestrator.py#L606-L612